### PR TITLE
@1aurabrown => Push notifications

### PR DESCRIPTION
**Pull request by alloy**
Getting back on that push notification train!

### CODE REVIEW

File: Artsy/Classes/View Controllers/AROnboardingViewController.m [hunk 1]
```diff
@@ -334,16 +334,21 @@ -(void) resetBackgroundImageView:(BOOL)animated completion:(void (^)(void))compl
     }];
 }
 
-- (void)dismissOnboardingWithVoidAnimation:(BOOL)createdAccount
+- (void)dismissOnboardingWithVoidAnimation:(BOOL)createdAccount;
+{
+    [self dismissOnboardingWithVoidAnimation:createdAccount didCancel:NO];
+}
+
+- (void)dismissOnboardingWithVoidAnimation:(BOOL)createdAccount didCancel:(BOOL)cancelledSignIn;
```
Comment (1aurabrown) [inline Artsy/Classes/View Controllers/AROnboardingViewController.m]: bad copy paste? this and the one above have the semicolon at the end

Comment (alloy) [inline Artsy/Classes/View Controllers/AROnboardingViewController.m]: Nope, it’s how my vim is setup. Always adding the semicolon makes it easy to copy-paste between interface and implementation.

This was discussed and ok’d when I started, so the code-base should already be littered with them :D

Comment (1aurabrown) [inline Artsy/Classes/View Controllers/AROnboardingViewController.m]: oh really, i've never seen that before and it looks weird to me, but whatever.

**Comment by 1aurabrown** [comment]
something failing

**Comment by alloy** [comment]
Oops.

File: Artsy/Classes/View Controllers/AROnboardingViewController.m (hunk 1 above)
Comment (alloy) [inline Artsy/Classes/View Controllers/AROnboardingViewController.m]: Yeah it looked weird to me as well for the longest time, it’s more a matter of pragmatism. Adding/removing a semicolon seems like inefficient ceremony to me.

**Comment by alloy** [comment]
Btw, this needs to be merged before it will work correct in beta/app store builds https://github.com/artsy/gravity/pull/8982

File: Artsy/Classes/ARAppDelegate.h [hunk 2]
```diff
@@ -16,7 +16,9 @@
 
 - (void)showTrialOnboardingWithState:(enum ARInitialOnboardingState)state andContext:(enum ARTrialContext)context;
 
-- (void)finishOnboardingAnimated:(BOOL)animated;
+// A sign-in is considered cancelled when the user taps the close button on a ARSignUpActiveUserViewController,
```
Comment (orta) [inline Artsy/Classes/ARAppDelegate.h]: `///` not `//`

**Pull request merged**